internal/corazawaf/rule.go

```diff
@@ -176,8 +176,8 @@ func (r *Rule) doEvaluate(phase types.RulePhase, tx *Transaction, cache map[tran
 
 	var matchedValues []types.MatchData
 	// we log if we are the parent rule
-	tx.DebugLogger().Debug().Str("tx_id", tx.id).Int("rule_id", rid).Msg("Evaluating rule")
-	defer tx.DebugLogger().Debug().Str("tx_id", tx.id).Int("rule_id", rid).Msg("Finish evaluating rule")
+	tx.DebugLogger().Debug().Int("rule_id", rid).Msg("Evaluating rule")
```

Comment:
Would it make sense to add rule_id also to the logging context instead of logging it with every log statement ? or is that something not easily doable ?

Reply:
Just added, looks alright.